src/main/resources/assets/stylesheets/cvs-version-1.hbs

```diff
@@ -200,7 +201,8 @@ vertical-align: top
 }
 
 .header__three-columns-two {
-width: 120.3333pt;
+min-width: 120.3333pt;
```

Comment:
It looks like this CSS is used when generating both the fail and VTG30 certificates (not only in Pass and PassNoSeatbelts). Have you checked those scenarios to see what the certficate looks like?

Reply:
Yes, it has it all on one line e.g. 9,999,999 kilometres